src/pkg/cli/client/byoc/baseclient.go

```diff
@@ -15,12 +15,13 @@ import (
 )
 
 const (
-	// Changing this will cause issues if two clients with different versions are using the same account
-	CdImageTag   = "public-beta"
-	CdTaskPrefix = "defang-cd" // WARNING: renaming this practically deletes the Pulumi state
+	CdTaskPrefix      = "defang-cd"   // WARNING: renaming this practically deletes the Pulumi state
+	CdDefaultImageTag = "public-beta" // for when a project has no cd version, this would be a old deployment
+	CdLatestImageTag  = "public-beta" // Update this to the latest CD service major version number whenever cd major is changed
```

Comment:
```suggestion
	CdDefaultImageTag = "public-beta" // for when a project has no cd version, this would be a old deployment
	CdLatestImageTag  = "public-beta" // Update this to the latest CD service major version number whenever cd major is changed
	CdTaskPrefix      = "defang-cd"   // WARNING: renaming this practically deletes the Pulumi state
```